assets/src/components/editing/toolbar/FullScreenModal.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { PropsWithChildren } from 'react';
 
 type Props = PropsWithChildren<{
-  children: JSX.Element;
+  children: React.ReactNode;
```

Comment:
This lets us have multiple children of FullScreenModal so you don't need an extra container element
```
<FullScreenModal>
  <button>1</button> 
  <button>2</button>
</FullScreenModal>
```

instead of

```
<FullScreenModal>
<div>
  <button>1</button> 
  <button>2</button>
</div>
</FullScreenModal>
```